redis/spring-cloud-starter-stream-sink-redis/src/main/java/org/springframework/cloud/stream/app/redis/sink/RedisSinkConfiguration.java

```diff
@@ -43,6 +45,7 @@
 @Configuration
 @EnableConfigurationProperties(RedisSinkProperties.class)
 @EnableBinding(Sink.class)
+@Import(SpelExpressionConverterConfiguration.class)
```

Comment:
We don't need this one finally!
See `@EnableBinding` 's `@Import` list
 :smile: